src/pages/instances/InstanceList.tsx

```diff
@@ -249,80 +254,76 @@ const InstanceList: FC = () => {
     }
   }, [instances]);
 
-  const getHeaders = (hiddenCols: string[]) =>
-    [
-      {
-        content: NAME,
-        sortKey: "name",
-        style: { width: `${COLUMN_WIDTHS[NAME]}px` },
-      },
-      {
-        content: TYPE,
-        sortKey: "type",
-        style: { width: `${COLUMN_WIDTHS[TYPE]}px` },
-      },
-      ...(isAllProjects
-        ? [
-            {
-              content: PROJECT,
-              sortKey: "project",
-              style: { width: `${COLUMN_WIDTHS[PROJECT]}px` },
-            },
-          ]
-        : []),
-      ...(isClustered
-        ? [
-            {
-              content: CLUSTER_MEMBER,
-              sortKey: "member",
-              style: { width: `${COLUMN_WIDTHS[CLUSTER_MEMBER]}px` },
-            },
-          ]
-        : []),
-      {
-        content: MEMORY,
-        style: { width: `${COLUMN_WIDTHS[MEMORY]}px` },
-      },
-      {
-        content: FILESYSTEM,
-        style: { width: `${COLUMN_WIDTHS[FILESYSTEM]}px` },
-      },
-      {
-        content: DESCRIPTION,
-        sortKey: "description",
-        style: { width: `${COLUMN_WIDTHS[DESCRIPTION]}px` },
-      },
-      {
-        content: IPV4,
-        className: "u-align--right",
-        style: { width: `${COLUMN_WIDTHS[IPV4]}px` },
-      },
-      {
-        content: IPV6,
-        id: "header-ipv6",
-        style: { width: `${COLUMN_WIDTHS[IPV6]}px` },
-      },
-      {
-        content: SNAPSHOTS,
-        sortKey: "snapshots",
-        className: "u-align--right",
-        style: { width: `${COLUMN_WIDTHS[SNAPSHOTS]}px` },
-      },
-      {
-        content: STATUS,
-        sortKey: "status",
-        className: "status-header status",
-        style: { width: `${COLUMN_WIDTHS[STATUS]}px` },
-      },
-      {
-        "aria-label": "Actions",
-        className: classnames({ "u-hide": panelParams.instance }),
-        style: { width: `${COLUMN_WIDTHS[ACTIONS]}px` },
-      },
-    ].filter(
-      (item) =>
-        typeof item.content !== "string" || !hiddenCols.includes(item.content),
-    );
+  const HEADERS: MainTableHeader[] = [
```

Comment:
```suggestion
  const headers: MainTableHeader[] = [
```
Better aligned with code style in the project.